schema/config_schema_v3.9.json

```diff
@@ -185,18 +230,21 @@
 
         "external_links": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
+        "group_add": {
+          "type": "array",
+          "items": {
+            "type": ["string", "number"]
+          },
+          "uniqueItems": true
+        },
```

Comment:
Need to review how these (should) interact with `user`; `user` can have `user:group`, which sets the primary group, and this sets auxiliary groups. For swarm services we renamed this to `groups`

```
--group list  Set one or more supplementary user groups for the container
```